### PR TITLE
[ZH] Add and update articles in `Storyboard/Scripting`

### PR DESCRIPTION
**Pull request by CloneWith**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Storyboard/Scripting/Commands/zh.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+- (end_r),(end_g),(end_b) 对应 endtime 时的颜色
+
+举个例子，要让对象作为阴影（全黑）出现，然后淡入原有颜色：
+
+```
+Sprite,Pass,Centre,"Sample.png",320,240
+_C,0,58810,59810,0,0,0,255,255,255
+```
+
+要让对象显示为黄色 (`#CCCC00`)：
```
Comment (agatemosu) [inline wiki/Storyboard/Scripting/Commands/zh.md:242]:
```suggestion
要让对象显示为黄色 (`#cccc00`)：
```

File: wiki/Storyboard/Scripting/General_Rules/zh.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+- 前景层
+
+注意与“设计”标签页中所示不同，“失败”与“通过”层不会同时出现。
+
+默认情况下，谱面指定的预览背景（可在[选歌界面](/wiki/Client/Interface#选歌界面)中看到的背景）置于所有四层下面。然而，如果相同的背景文件被引用为故事板中的对象，则在谱面加载后会立即消失。通常会将谱面的预览背景图指定为（时间与精灵图上）第一个对象，并使用“渐隐”（变亮）命令将背景“引进”给玩家。
+
+#### 重叠规则
+
+- 在**不同**层重叠的对象会按上述顺序绘制（例如，位于前景层的所有对象都会显示在其他三层上面）。
+- 在**同一**层重叠的对象会按指定的顺序绘制（例如，如果对象 1 先在 .osb 或 .osu 文件中被指定，然后是对象 2，二者处于同一层，则对象 2 会显示在对象 1 上层）。
```
Comment (YumeMuzi) [inline wiki/Storyboard/Scripting/General_Rules/zh.md:52]:
```suggestion
- 在**同一**图层重叠的对象会按指定的顺序绘制（例如，如果对象 1 先在 .osb 或 .osu 文件中被指定，其后是对象 2，二者处于同一图层，则对象 2 会显示在对象 1 上层）。
```

其他`层`也要改成`图层`

File: wiki/Storyboard/Scripting/General_Rules/zh.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+使用故事板而非视频文件的想法，来自**动态更改元素，使之与游玩情况匹配的能力。**取决于玩家的表现状况，osu! 一次只会显示失败或通过层之一。这两个状况被称作“失败状态”与“通过状态”。
+
+**游玩时间前**（如第一个[圆圈/滑条/转盘](/wiki/Gameplay/Hit_object)前，不一定在 MP3/OGG 音频开始播放前）的状态:
+
+- 总处于通过状态，不会显示失败层。不推荐在这里使用通过或失败层，因为在此时认为玩家“通过”毫无意义。
+
+**游玩期间**（“掉血时间”，玩家需要点击物件，防止血量下降）的状态:
+
+- 当处于第一个颜色连击，或前一个连击末尾获得“激”（或“精英节拍！”，连击中全部获得 300）时，处于通过状态。
+- 否则为失败状态。注意与 DS 游戏（有三个状态）不同，“喝”（或“节拍！”）判定没有对应的状态。
```
Comment (YumeMuzi) [inline wiki/Storyboard/Scripting/General_Rules/zh.md:66]:
```suggestion
- 除了以上的情况之外，均为失败状态。注意与 DS 游戏（有三个状态）不同，“喝”（或“击中！”（EBA 中的判定））判定没有对应的状态。
```

File: wiki/Storyboard/Scripting/General_Rules/zh.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+使用故事板而非视频文件的想法，来自**动态更改元素，使之与游玩情况匹配的能力。**取决于玩家的表现状况，osu! 一次只会显示失败或通过层之一。这两个状况被称作“失败状态”与“通过状态”。
+
+**游玩时间前**（如第一个[圆圈/滑条/转盘](/wiki/Gameplay/Hit_object)前，不一定在 MP3/OGG 音频开始播放前）的状态:
+
+- 总处于通过状态，不会显示失败层。不推荐在这里使用通过或失败层，因为在此时认为玩家“通过”毫无意义。
+
+**游玩期间**（“掉血时间”，玩家需要点击物件，防止血量下降）的状态:
+
+- 当处于第一个颜色连击，或前一个连击末尾获得“激”（或“精英节拍！”，连击中全部获得 300）时，处于通过状态。
```
Comment (YumeMuzi) [inline wiki/Storyboard/Scripting/General_Rules/zh.md:65]:
```suggestion
- 当处于第一个颜色连击，或前一个连击末尾获得“激”（或“精英节拍！”（EBA 中的判定，连击中全部获得 300））时，处于通过状态。
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**